collect_app/src/main/res/values/typography.xml

```diff
@@ -36,4 +36,8 @@
         <item name="android:textAllCaps">false</item>
         <item name="android:letterSpacing">0.04</item>
     </style>
+
+    <style name="TextAppearance.Collect.Link" parent="TextAppearance.MaterialComponents.Body2">
```

Comment:
Again, this should make it easier to keep track of the color customization on the views when we switch to better colors.